packages/router/src/grpc/view-protocol-server/swap-by-commitment.ts

```diff
@@ -1,26 +1,34 @@
 import type { Impl } from '.';
 import { servicesCtx } from '../../ctx';
+import { watchSubscription } from './util/watch-subscription';
 
 import { SwapRecord } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/view/v1alpha1/view_pb';
 
 import { ConnectError, Code } from '@connectrpc/connect';
 
-export const swapByCommitment: Impl['swapByCommitment'] = async (req, ctx) => {
+export const swapByCommitment: Impl['swapByCommitment'] = async (
+  { swapCommitment: findSwapCommitment, awaitDetection },
```

Comment:
OK if we leave this name as it was? It seems like an arbitrary rename, and the new name sounds more like a function than like a message object.